qt/scientific_interfaces/Indirect/ConvFitModel.cpp

```diff
@@ -715,6 +611,25 @@ void ConvFitModel::setParameterNameChanges(
       model, backgroundIndex, m_temperature.is_initialized());
 }
 
+std::vector<std::pair<std::string, int>>
+ConvFitModel::getResolutionsForFit() const {
+  std::vector<std::pair<std::string, int>> resolutionVector;
+  resolutionVector.reserve(20);
```

Comment:
Why 20?

Reply:
Just that there are normally around 20 and no more items added to this vector, but as it is not know exactly may as well leave it to the compiler so I'll remove as the vector should never get large is unlikely to be much of a performance hit.